boss-test.py

```diff
@@ -0,0 +1,117 @@
+import requests
```

Comment:
Let's leave this file out of the PR since we won't be pushing boss code to the master branch for a while.